amida-tech/hl7-telescoper-api-server
# Hl7 13 am get file endpoint route

Ticket: https://jira.amida.com/browse/HL7-13

## What does this ticket do?
- Adds endpoint to retrieve uploaded HL7 files
- Adds unit test for retrieval ruction
- Also removed the date from filename in order to retrieve filenames without having to know timestamp


## How to Test:
 -  Send a GET request from postman with the filename in the request URL
 - Make sure the contents of the file are outputted 
- Make sure `yarn test` passes and there are no linting errors. 


## NEW TICKET

So I decided to create a new ticket to update the endpoint to what Perry said about returning an Object as opposed to just the raw HL7 file. So check that out as well

New Ticket=> https://jira.amida.com/browse/HL7-19
